content/docs/gke/gcp-e2e.md

```diff
@@ -767,26 +681,26 @@ Follow these steps to build an image from your code:
     cd ${WORKING_DIR}
     ```
 
-1. Set the path in [Container Registry][container-registry] to push the
+2. Set the path in [Container Registry][container-registry] to push the
```

Comment:
Please revert this change, and all the similar changes from `1.` to a specific number below. We keep the list formatting as just `1.` to avoid having to renumber when we add/remove steps.